tests/components/shelly/test_sensor.py

```diff
@@ -56,26 +56,61 @@
 DEVICE_BLOCK_ID = 4
 
 
+@pytest.mark.parametrize(
+    ("entity", "initial_state", "block_id", "attribute", "value", "unique_id"),
+    [
+        (
+            "test_name_channel_1_power",
```

Comment:
I think it would be better to add a dedicated test for these sensors and not reuse the existing test.
The existing test cover a sensor that has a channel and this test is for sensors which are not tied to a channel.
We should also cover all states and unknown state same as https://github.com/home-assistant/core/pull/140597 so it would be better to add a new test or reuse the test added there and not this test.

Reply:
Done